appengine/genetics/js/genetics.js

```diff
@@ -29,48 +29,36 @@ goog.provide('Genetics');
 goog.require('BlocklyDialogs');
 goog.require('BlocklyGames');
 goog.require('BlocklyInterface');
-goog.require('Genetics.Blocks');
 goog.require('Genetics.Cage');
 goog.require('Genetics.Mouse');
 goog.require('Genetics.MouseAvatar');
 goog.require('Genetics.Visualization');
-goog.require('Genetics.soy');
 goog.require('goog.array');
-goog.require('goog.ui.Tab');
-goog.require('goog.ui.TabBar');
-
 
 /**
- * The name of the app.
- * @const {string}
+ * Optional callback function for when a game ends.
+ * @param {boolean} success Whether the game/level was succeeded.
+ * @type {function(boolean)}
  */
-BlocklyGames.NAME = 'genetics';
+Genetics.endGame = null;
```

Comment:
If this is a callback, I'd expect a name that's more like `onEndGame` -- since this doesn't actually end the game, it's an additional thing to do when the game ends.